integrations/webchat/examples/using-carbon/client/javascript/.proxyrc

```diff
@@ -0,0 +1,5 @@
+{
+  "/api": {
+    "target": "http://localhost:3001/"
+  }
+}
```

Comment:
Whats this for? Just curious :)

Reply:
Oh, I have no idea. Ethan added this in the template and I probably should have removed it from here. It's presumably for examples that have a back-end server they need to call.